website/docs/d/workspace.html.markdown

```diff
@@ -35,6 +35,7 @@ In addition to all arguments above, the following attributes are exported:
 * `id` - The workspace ID.
 * `allow_destroy_plan` - Indicates whether destroy plans can be queued on the workspace.
 * `auto_apply` - Indicates whether to automatically apply changes when a Terraform plan is successful.
+  `drift_detection` - Indicates whether drift detection is enabled for the workspace.
```

Comment:
Let's update the docs since the attribute name has changed:

```suggestion
  `assessments_enabled` - Indicates whether health assessments such as drift detection are enabled for the workspace.
```